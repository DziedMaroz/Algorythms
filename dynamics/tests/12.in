9890648190679
565
424157331369
2501765834
56
3
9959411
7530493
06
4
1753115568272
754931
330
950595243078
123480702195
085827158758
25614
2
1
7
3723210
93
07785
89550249381
08
4752936652665
8890868823160
419368803
369495708
920930170
839304198080
91127066
414745453566
62820062
5430928094062
295
80692972
8807934
28
404
63154
07788353796
91
313922
2259376078
2
20617519
8575894028
440050
4650666053
0094082975
320
684
744048267
45157249587
2420036
96699501
33159
904259470129
684
1
175
369742
87409639575
908106
1247
14173396
616290
46717
7929
4591
15974564
6633996017
778629261226
96229589937
50088007
882
31
34621
31057393
33
9751113
40245
43
228314342312
2
39
557
0
89295
474132714445
77598
35912
12263
83
6081282241257
837602363426
217
405031457022
79
008967
162844113
49
2933895870558
8783
4
8
02
89443409773
0132
47711466217
516
51279144017
9
50364993780
2857
4
7654
6236904031
3813
5323699
247
336728693888
4979052152
00976
061844
85810436
644
51006294
8723434014
45400
62846
227323594
857
660
51562039
77860
66395028
88908
4
731118
62
8294
151333
563950824659
083560806
2153426791448
7
3691207692
433
681826415
712754032651
732254917473
52246492308
0560474
59036203306
010
1630549668853
1742943
1
773266
9
92475
8972173
0791253
837459
067893508
16878
395
392
66542
5
260933418
971738
13535
5414287
5143412010
1779
024709
3727
39238
66954
76
88
34285437077
934203
6790387788738
902960
53693
5568769676
4295
5721
63
016631045900
0
9276844
07
28
9265915798318
7136963637238
295870508
248
0592063947667
22
3542451682
7
6
008
755
3
741796569
91
2758596532693
694
95070
23
1597084
4
9102983404911
69470278792
449806706925
7771278946032
3431
126722399
9
71469
46182832734
982068301
7036
108756608698
3
085345352
77018505474
4374979908801
2028820540
053
2
7
3702685709
8924
49876627
1150708223130
5722280837864
08692
86795893734
5461538795
7730151055848
9523
893882
440893
12805183
6729
0
69
5
15328
1261
1615
003
596
93025
8526544071
802527173
839
2462105608
63
3035783
4
054028470
3760542145548
533203453388
672569
6781201751462
1
60
004885608
0
44370606
40
87686
555799891967
731012
1593
599199266
994184427508
80
74211
9
5615459
32
415
224
32531
407
359624592489
0
326505
0066
7992272132
68419582403
26
70
1862433
676673964504
305598107474
419
4851120168133
445049205457
3638
9748033789515
6641009488
241158014
882558881
1
603
9910
2087128310210
5
1
7124
92768
0871084
1
4332
7075
26024016
0657238031742
224
4774519553494
29877
6431738401
7260420168369
863486013
3
516867
075484
750671991
6
663
520113305032
4396564612387
981
83883
20780098408
132
746
909
816
5462009738
00111379
597
8218056
342001343412
745794122
782010910271
3692194363626
959350
035431654192
6951460586
53994422
5640757
32764
510249115970
334497
780676
4670976
9142
18378
10570726632
54997429438
2
091798
73815918
2994817947346
996592147124
99270
89141
537771707804
87694
02903
18
74328814608
149501510
328635
5342
850003
60612652534
4070
13757670
6995396
35365
189406
55720
57086418
0724794742947
4594698
19146141
8841473
0777932542014
6481306838
0113
5622717
929777
24237299
416631103
95052202
09811498
289
4371992868903
553600187
71593
807
684337547
1408370
4745
731402002
2283100586382
96782543
1432
611807082
1
7369453120026
8344371
2078794
10130
11158307
770736
18557
30114
775
913269
51608
38566643
2
483121717
35165245
300750
821
995
89
31636972
0
4
44
203853
72310279999
9776174748
4459101786
5427384906904
814
912316826
8406900489005
52
27530292
4208378571
5697381400
74480696
047630590743
775
6540426595
9833032307
337790809
96660950
791628
81
93105
92972
3039086956
77161022786
5
794721277
33
88197283
97524655491
3653
85
10523048
795496024792
7257777382
1349
88
85521728129
79
99380
667
52558187184
0534924751
5941306362
821465
9943
1315662311
07140108
80905030099
054
192746825
30
0961433906949
355
9370586084869
357127489
86776676498
046232012357
97
592262878371
1
771581
545
04
591862449424
011
861930615
4
627326
01
4503695849408
89724331064
37172
42528146403
279901
0118
968777581
947161
145742111958
9676717696504
005412
25149261
22
19060
0202275
40656731
04863978
499
3101158267
257010
842700706501
0
625410
5
97133361
087863244427
76
3037095
210539468
9396550379
597452
929
694927826
8403
53
83121
131
01
6617
3527905
9
31149036046
777155839622
700687557
4304563
899349
6362555
6885
484990040
754
713
13185
8
331272425
