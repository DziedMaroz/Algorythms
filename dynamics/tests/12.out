8
9 89 06 4 81 9 06 79
